pkgs/by-name/gz/gz-cmake/package.nix

```
@@ -25,6 +31,26 @@ stdenv.mkDerivation (finalAttrs: {
     pkg-config
   ];
 
+  cmakeFlags = [
+    (lib.cmakeBool "BUILDSYSTEM_TESTING" finalAttrs.doCheck)
+  ];
+
+  nativeCheckInputs = [ python3 ];
+
+  # 98% tests passed, 1 tests failed out of 44
+  # 44 - c_child_requires_c_nodep (Failed)
+  #
+  # Package gz-c_child_private was not found in the pkg-config search path.
+  # Perhaps you should add the directory containing `gz-c_child_private.pc'
+  # to the PKG_CONFIG_PATH environment variable
+  # No package 'gz-c_child_private' found
+  doCheck = false;
```

```suggestion
  postPatch = ''
    patchShebangs examples/test_c_child_requires_c_no_deps.bash
    substituteInPlace examples/CMakeLists.txt --replace-fail \
      "$""{CMAKE_INSTALL_LIBDIR}" "lib64"
  '';
  
  doCheck = true;
```

Seems OK for me with this

---

It fails on `{aarch64, x86_64}-darwin`; is there some `stdenv.hostPlatform.$VAR` that sets this value for us?

---

fixed (at least aarch64-darwin) for in #406744. I don't have a x86_64-darwin to check